gpustack/worker/backends/custom.py

```diff
@@ -118,6 +118,7 @@ def _create_workload(
         workload_plan = WorkloadPlan(
             name=self._workload_name,
             host_network=True,
+            shm_size=10 * 1 << 30,  # 10 GiB
```

Comment:
How do we pick this value, is it safe enough?

Reply:
shm_size is a limitation value, 10GiB is good enough for most scenarios.